app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Consider your application asks for confirmation of email.
![image](https://user-images.githubusercontent.com/53402600/194402706-ea5ccbd5-59c7-4ec5-bbdc-9bdbfe031fdd.png)